forecasters/aardvark/R/modeling.R

```diff
@@ -317,12 +291,49 @@ make_cv_glmnet <- function(alpha = 1, fdev = 0, mnlam = 100, n_folds = 10){
 }
 
 make_predict_glmnet <- function(lambda_choice){
-  # Closure, allowing us to pass parameters to predict.glmnet.
   # Inputs:
   #   lambda_choice: either "lambda.1se" or "lambda.min"
   predict_glmnet <- function(fit, X, offset, ...){
-    stopifnot(is.character(lambda_choice))
     preds <- predict(fit, newx = X, newoffset = offset, s = lambda_choice)[,1]
     return(preds)
   }
 }
+
+#' @importFrom covidcast aggregate_signals
+#' @import evalcast
+long_to_wide <- function(df){
+  # Manipulate evalcast df to the wide format previously used during evalforecast era
+  # This is a really hacky way to circumvent the issue while GitHub issue #269 is pending
+  if ( nrow(unique(df %>% select(data_source, signal, geo_value, time_value))) < nrow(df) ){
```

Comment:
Do you still need this? `evalcast` can return a signals list or in wide format. I see that 269 is still open.